src/Language/PureScript/AST/Exported.hs

```diff
@@ -144,3 +141,16 @@ isDctorExported ident (Just exps) ctor = test `any` exps
   test (TypeRef _ ident' Nothing) = ident == ident'
   test (TypeRef _ ident' (Just ctors)) = ident == ident' && ctor `elem` ctors
   test _ = False
+
```

Comment:
> I still think having an actual document structure derived from the export section would be a nice way to go about extracting documentation.

The problem is, we need the `desugarImports` part to have run before we can start getting useful information out of it, because of things like modules without export lists, or export lists which use `Foo(..)` to export all the data constructors, or so on. Of course it would be possible to handle those things too, but at that point we're basically reimplementing `Sugar.Names`.